windows_containers/wmco_rn/windows-containers-release-notes-10-17-x-prereqs.adoc

```diff
@@ -4,7 +4,14 @@
 = Windows Machine Config Operator prerequisites
 include::_attributes/common-attributes.adoc[]
 
-The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
+The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator (WMCO). See the vSphere documentation for any information that is relevant to only that platform.
+
+[id="wmco-prerequisites-supported-install-10.17.0_{context}"]
+== WMCO supported Installation method
+
+The WMCO fully supports installing Windows nodes into installer-provisioned infrastructure (IPI) clusters. This is the preferred {product-title} installation method.
+
+For user-provisioned infrastructure (UPI) clusters, the WMCO supports installing Windows nodes only into a UPI cluster installed with the `platform: none` field set in the `install-config.yaml` file (bare-metal or provider-agnostic) and only for the xref:../../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[BYOH (Bring Your Own Host)] use case. UPI is not supported for any other platform.
```

Comment:
```suggestion
For user-provisioned infrastructure (UPI) clusters, the WMCO supports installing Windows nodes only in the following cases:

* In a UPI cluster installed with the `platform: none` field set in the `install-config.yaml` file (bare-metal or provider-agnostic)
* For the xref:../../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[BYOH (Bring Your Own Host)] use case

UPI is not supported for any other platform.
```
I tried to make the information in the first sentence easier to parse.

Reply:
This seems to make it an either/or scenario, rather than both conditions are required.